src/main/java/nl/tudelft/dnainator/ui/views/AbstractView.java

```diff
@@ -0,0 +1,141 @@
+package nl.tudelft.dnainator.ui.views;
+
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Bounds;
+import javafx.geometry.Point2D;
+import javafx.scene.Group;
+import javafx.scene.layout.Pane;
+import javafx.scene.transform.Affine;
+import javafx.scene.transform.NonInvertibleTransformException;
+import javafx.scene.transform.Scale;
+import javafx.scene.transform.Transform;
+import javafx.scene.transform.Translate;
+import nl.tudelft.dnainator.ui.widgets.dialogs.ExceptionDialog;
+
+import java.io.IOException;
+
+/**
+ * An abstract class for common functionality needed by all the views.
+ */
+public abstract class AbstractView extends Pane {
+	protected static final double SCALE = .1;
```

Comment:
No need for this to be final. If it isn't, you can remove some code from PhylogeneticView that uses its own `SCALE`. See the comments at these lines.